archive/p/python/lcs.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+import argparse
+
+def input_list(list_str):
+
+    return [int(x.strip(" "), 10) for x in list_str.split(',')]
```

Comment:
By the way, I think this is really, really cool! I've used argparse in the past, but I didn't realize you could create your own input types. I was over hear looking through docs for `input_list` until I saw: 

```
type= can take any callable that takes a single string argument and returns the converted value:
```

Cool stuff!